package.json

```diff
@@ -190,8 +190,8 @@
     "core-js": "^2.5.7",
     "jquery": "^3.3.1",
     "prop-types": "^15.6.0",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
+    "react": "^16.5.2",
+    "react-dom": "^16.5.2",
```

Comment:
Missing yarn.lock, was this updated with npm? Is it needed for React.Fragments, right?

Reply:
No and yes, fixing now